docs/source/pillars/computing_technology.md

```diff
@@ -165,9 +224,6 @@ _The ability of the TRE operator to administer and secure network infrastructure
   - Network infrastructure must prevent unauthorised access to resources on the network.
     This may include firewalls, network segmentation, and restricting connections to the network.
   - Mandatory
-* - You must monitor the network configuration of your TRE to check for misconfigurations and vulnerabilities.
-  - This may include regular vulnerability scanning, and penetration testing.
-  - Mandatory
```

Comment:
This feels like an odd one to remove. How can we be confident a TRE is correctly configured, and secure, if we don't test that?

I think this covers more than just networking though.

Reply:
I agree.

Reply:
Reverted in 39cbbe0